src/openforms/forms/tests/test_import_export.py

```diff
@@ -1401,10 +1401,8 @@ class ImportObjectsAPITests(TempdirMixin, OFVCRMixin, TestCase):
 
     VCR_TEST_FILES = PATH / "files"
 
-    def setUp(self):
-        super().setUp()
-
-        self.objects_api_group = ObjectsAPIGroupConfig.objects.create(
+    def _create_objects_api_group(self):
+        return ObjectsAPIGroupConfig.objects.create(
```

Comment:
wouldn't it make more sense to define a factory with a trait for all this boilerplate? these helper methods in tests are quite annoying to track down/context switch to then infer what they're setting up

something like `ObjectsAPIGroupFactory.create(for_test_docker_compose=True)`